cmd/start-reverse-payment-proxy/main.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/rs/zerolog"
+	"github.com/statechannels/go-nitro/reverseproxy"
+	"github.com/urfave/cli/v2"
+)
+
+const (
+	NITRO_ENDPOINT  = "nitroendpoint"
+	PORT            = "port"
+	DESTINATION_URL = "destinationurl"
+)
+
+func main() {
+	app := &cli.App{
+		Name:  "start-reverse-payment-proxy",
+		Usage: "Runs an HTTP payment proxy that charges for HTTP requests",
+		Flags: []cli.Flag{
+			&cli.UintFlag{
+				Name:  PORT,
+				Usage: "Specifies the port to run the proxy on",
+				Value: 5511,
+			},
+			&cli.StringFlag{
+				Name:  NITRO_ENDPOINT,
+				Usage: "Specifies the endpoint of the Nitro RPC server",
+				Value: "localhost:4007/api/v1",
+			},
+			&cli.StringFlag{
+				Name:  DESTINATION_URL,
```

Comment:
It could be nice to use a consistent format when passing in the different adresses